packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -106,7 +106,7 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 	}
 
 	private getBlobContents(snapshotTree: ISnapshotTreeWithBlobContents) {
-		for (const [id, value] of Object.entries(snapshotTree.blobsContents)) {
+		for (const [id, value] of Object.entries(snapshotTree.blobsContents ?? {})) {
```

Comment:
These types became nullable with the module change - is this fix safe?

Reply:
I believe this is a safe fix. It won't change current behavior in case there are no blobContents.